apps/arweave/src/ar_fork.erl

```diff
@@ -94,3 +94,11 @@ height_2_6() ->
 			1132210. % Targeting 2023-03-06 14:00 UTC
 	-endif.
 -endif.
+
+-ifdef(FORKS_RESET).
+height_2_6_8() ->
+	0.
+-else.
+height_2_6_8() ->
+	1189720. % Targeting 2023-05-30 16:00 UTC
```

Comment:
I estimate ~1189560 computing the average block time over the 10000 blocks window (128 seconds).

Reply:
https://github.com/ArweaveTeam/arweave/pull/422/commits/15e10af482b0e9e58e95c43981386e182e7f2fdb